config/routes.rb

```diff
@@ -23,8 +23,8 @@
   end
 
   scope ":slug", module: "tenants", as: "tenant" do
-    get "/" => "items#index"
-    resources :items
+    get "/" => "projects#index"
+    resources :projects
```

Comment:
I would add an except :delete here (or only new, create, edit, update), since per the specs users are not allowed to delete projects, only retire them